### ISSUE TITLE
Docker in Docker no longer available on AKS 1.19+

### ISSUE THREAD

**Issue by rommelandrea** [MicrosoftDocs/azure-devops-docs#10342]
As reported here https://github.com/Azure/AKS/issues/1989, since version 1.19 of AKS docker is no longer available. Is there another way to be able to continue build docker images with Docker in Docker or similar?
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b31d4f6c-d16d-2ba1-1294-6aa94e328046
* Version Independent ID: 88aec599-4410-697d-1002-1c41b1c17bb6
* Content: [Run a self-hosted agent in Docker - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/docker?view=azure-devops)
* Content Source: [docs/pipelines/agents/docker.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/agents/docker.md)
* Product: **devops**
* Technology: **devops-cicd-agents**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by TimShererWithAquent** [comment]
@rommelandrea - Andrea - Thanks for your question. It looks like you're working through an issue with your scenario or implementation, rather than an issue with the documentation. Please consider these resources:

- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps Support Bot](https://azuredevopsvirtualagent.azurewebsites.net/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

**Comment by rommelandrea** [comment]
@TimShererWithAquent  I'm sorry, I think I haven't explained myself well enough. I'm referring to the doc and not implementation.

The documentation on which I opened the issue is no longer valid because, from Kubernetes version 1.19, it is no longer possible to use the functionality Docker in Docker because docker.sock no longer exists.

In the issue I linked there is the reason why the documentation is no longer valid.
